cron/monthly/host-settlement.ts

```diff
@@ -80,25 +81,46 @@ export async function run(baseDate: Date | moment.Moment = defaultDate): Promise
     },
   );
 
+  let slugs, skipSlugs;
+  if (process.env.SLUGS) {
+    slugs = process.env.SLUGS.split(',').map(str => str.trim());
+  }
+  if (process.env.SKIP_SLUGS) {
+    skipSlugs = process.env.SKIP_SLUGS.split(',').map(str => str.trim());
+  }
+
   for (const host of hosts) {
     if (HOST_ID && host.id !== parseInt(HOST_ID)) {
       continue;
     }
+    if (slugs && !slugs.includes(host.slug)) {
+      continue;
+    }
+    if (skipSlugs && skipSlugs.includes(host.slug)) {
+      continue;
+    }
 
-    const pendingPlatformTips = await getPendingPlatformTips(host, { status: ['OWED'], endDate });
-    const pendingHostFeeShare = await getPendingHostFeeShare(host, { status: ['OWED'], endDate });
+    let pendingPlatformTips, pendingHostFeeShare;
+    if (!KIND || KIND === PLATFORM_TIP_DEBT) {
+      pendingPlatformTips = await getPendingPlatformTips(host, { status: ['OWED'], endDate });
+    }
+    if (!KIND || KIND === HOST_FEE_SHARE_DEBT) {
+      pendingHostFeeShare = await getPendingHostFeeShare(host, { status: ['OWED'], endDate });
+    }
 
     const plan = await host.getPlan();
 
     let items = [];
 
+    const transactionsKinds = KIND ? [KIND] : [PLATFORM_TIP_DEBT, HOST_FEE_SHARE_DEBT];
+
     const transactions = await sequelize.query(
       `
       SELECT t.*
       FROM "Transactions" as t
       INNER JOIN "TransactionSettlements" ts ON ts."TransactionGroup" = t."TransactionGroup" AND t.kind = ts.kind
       WHERE t."CollectiveId" = :CollectiveId
-        AND t."kind" IN ('PLATFORM_TIP_DEBT', 'HOST_FEE_SHARE_DEBT')
+        AND t."kind" IN (${transactionsKinds.map(kind => `'${kind}'`).join(', ')})
```

Comment:
Even though there's no risk of SQL injection here I'd still advocate for using replacements, they let you easily copy-paste the query in DB tools and improve readability.

```suggestion
        AND t."kind" IN (:transactionsKinds)
```

Then:
```es6
replacements: { CollectiveId: host.id, endDate: endDate, transactionsKinds },
```

Reply:
Definitely!